lib/syskit/roby_app/plugin.rb

```diff
@@ -876,6 +934,7 @@ def self.setup_rest_interface(app, rest_api)
                 require "syskit/roby_app/rest_api"
                 rest_api.mount REST_API => "/syskit"
             end
+
```

Comment:
Stray line. Please execute rubocop before submitting code.

Reply:
I've executed rubocop and corrected all offenses pointed, except this one.
It implies that the arguments are not needed, but without them it would raise an error of "0 of 4 arguments expected"
![Screenshot_20210421_101629](https://user-images.githubusercontent.com/50235244/115560292-d1901780-a28a-11eb-9ed8-df09b790a639.png)

Reply:
Also, I've run rubocop in the TmpRootCA file and corrected most of offenses. There are a few remaining but they raise offenses as "Line is too long" and "Assingment Branch Condition size too high" (which I believe refers to the linecount of single methods). Do I correct them too or let them slide, been layout/style offenses?